recipes-graphics/vc-graphics/files/vchiq.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+
+# SPDX-FileCopyrightText: meta-raspberrypi contributors
```

Comment:
Let's drop the header for this file for now - I'd like to check its contribution first.